storage/store.go

```diff
@@ -568,7 +568,7 @@ func (s *Store) startGossip() {
 	ctx := s.Context(nil)
 	// Periodic updates run in a goroutine and signal a WaitGroup upon completion
 	// of their first iteration.
-	s.initComplete.Add(1)
+	s.initComplete.Add(2)
```

Comment:
I'm surprised this was a race. Don't both calls to `Add()` definitely happen before any call to `WaitForInit`?